sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Customized/Models/DataProtectionBackupVaultProperties.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+namespace Azure.ResourceManager.DataProtectionBackup.Models
+{
+    /// <summary> Backup Vault. </summary>
+    public partial class DataProtectionBackupVaultProperties
+    {
+        /// <summary> Is vault protected by resource guard. </summary>
+        public bool? IsVaultProtectedByResourceGuard { get; set; }
```

Comment:
This property is marked as `readOnly: true` in the API spec after our last stable release. So the latest generated code will remove the setter. I manually add setter back but we will not send it to service as it is removed from the serialization.